src/components/Link/ArrowLink/ArrowLink.module.scss

```diff
@@ -0,0 +1,78 @@
+@use '../../../styles/animations';
```

Comment:
I know this file was just pulled from `zNS-dapp`, but can you refactor to handle proper SCSS nesting and convert `px` to `rem`?

Reply:
Never checked this scss file